From 423ac27a9402ecd2a55409869d727d11b8a6baf2 Mon Sep 17 00:00:00 2001
From: Meng Dongyang <daniel.meng@rock-chips.com>
Date: Mon, 5 Mar 2018 17:02:07 +0800
Subject: [PATCH] arm64: dts: rockchip: px30: modify error of EHCI and OHCI DT

Modify the clock name of EHCI and OHCI controllers, add
property of "status" for OHCI.

Change-Id: I444a906bc26e26989f5f6011de949b816266b9c6
Signed-off-by: Meng Dongyang <daniel.meng@rock-chips.com>
---
 arch/arm64/boot/dts/rockchip/px30.dtsi | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/arch/arm64/boot/dts/rockchip/px30.dtsi b/arch/arm64/boot/dts/rockchip/px30.dtsi
index 8d86d7e6cc53..35c9de4730ba 100644
--- a/arch/arm64/boot/dts/rockchip/px30.dtsi
+++ b/arch/arm64/boot/dts/rockchip/px30.dtsi
@@ -941,7 +941,7 @@
 		reg = <0x0 0xff340000 0x0 0x10000>;
 		interrupts = <GIC_SPI 60 IRQ_TYPE_LEVEL_HIGH>;
 		clocks = <&cru HCLK_HOST>, <&u2phy>;
-		clock-names = "usbhost", "arbiter", "utmi";
+		clock-names = "usbhost", "utmi";
 		power-domains = <&power PX30_PD_USB>;
 		phys = <&u2phy_host>;
 		phy-names = "usb";
@@ -953,10 +953,11 @@
 		reg = <0x0 0xff350000 0x0 0x10000>;
 		interrupts = <GIC_SPI 61 IRQ_TYPE_LEVEL_HIGH>;
 		clocks = <&cru HCLK_HOST>, <&u2phy>;
-		clock-names = "usbhost", "arbiter", "utmi";
+		clock-names = "usbhost", "utmi";
 		power-domains = <&power PX30_PD_USB>;
 		phys = <&u2phy_host>;
 		phy-names = "usb";
+		status = "disabled";
 	};
 
 	gmac: ethernet@ff360000 {
-- 
2.35.3

